### PR TITLE
feat: bEAC-209 - UI for Dial-in Audio

### PR DESCRIPTION
**Pull request by alucio-echen**
See description for: https://github.com/alucioinc/eeb/pull/104 for more information

### CODE REVIEW

**Comment by chibchombiano26** [comment]
<a href="https://www.loom.com/share/11bfdcae98254451b6ad5f7d62e15140"> <p>Beacon - Watch Video</p>  </a>

seems like the call ask for an enviroment number that we are not showing in the modal

**Comment by chibchombiano26** [comment]
also seems like the phone indicator is missing:

<a href="https://www.loom.com/share/904301356061485398899c37b10d432a"> <p>Beacon - Watch Video</p>  </a>

**Comment by chibchombiano26** [comment]
mostly LTM i think, we can address the issues as a bugs, or not sure if you want to addressed those changes in this ticket either way is good for me

**Review by chibchombiano26 — COMMENTED** [review]
mostly LGTM I think, we can address the issues as a bug, or not sure if you want to address those changes in this ticket, either way, is good for me

File: alucio/components/Virtual/DialInModal/DialInModal.tsx [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+  localPhoneAudio: boolean,
+}
+
+function DialInModal(props:DialInModalProps) {
+  const { accessCode, dialInNumber, isShown, onClose, localPhoneAudio } = props;
+  const modalHeaderText = localPhoneAudio ? 'Use Computer Audio' : 'Call-In Instructions'
+
+  return (
+    <Modal visible={isShown} backdropStyle={styles.backdrop}>
+      <View style={styles.modalWrapper}>
```
[the remark this answers is not in the document]
Comment (alucio-echen) [reply alucio/components/Virtual/DialInModal/DialInModal.tsx:74]: Let's save this for the UI refactor of virtual

File: alucio/components/AudioLevelIndicator/AudioLevelIndicator.tsx [hunk 2]
```diff
@@ -27,15 +27,23 @@ function AudioLevelIndicator({
...
   audioTrack?: AudioTrack;
   background?: string;
+  isPhoneAudio: boolean;
 }) {
-  const SIZE = size || 24;
+  const SIZE = size || 16;
   const ref = useRef<SVGRectElement>(null);
   const mediaStreamRef = useRef<MediaStream>();
   const isTrackEnabled = useIsTrackEnabled(audioTrack as LocalAudioTrack | RemoteAudioTrack);
+  /* eslint-disable max-len */
```
Comment (alucio-echen) [inline alucio/components/AudioLevelIndicator/AudioLevelIndicator.tsx:41]: The phone icon looks like this: 

image

Comment (chibchombiano26) [reply alucio/components/AudioLevelIndicator/AudioLevelIndicator.tsx:41]: the icon looks good, but probably a top margin, will be good

**Review by chibchombiano26 — APPROVED** [review]
LGTM Great wWork

**Pull request merged**